PinvokeAnalyzer_UWPApis.txt

```diff
@@ -0,0 +1,2298 @@
+bcrypt.dll!BCryptCloseAlgorithmProvider
```

Comment:
Is this temporary until buildtools is updated, or is the plan that the corresponding lists are removed from buildtools?